assembl/static2/js/app/components/cookies/acceptCookiesModal.spec.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
strange. I don't need to add this disable (see `postActions.spec.jsx` for example)

Reply:
This is a copy paste, I probably don't need it here indeed since I am not in the tests folder

Reply:
Actually I do get an eslint error (enzyme not being listed as dep) here